inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -43,6 +43,19 @@ def __init__(self, protein_pdb: str, temperature: float = 300):
         self.protein_pdb = files.resolve_path(protein_pdb)
         self.temperature = temperature
 
+    def set_default_output_files(self):
+        """Set the downloadable files as the compressed trajectory adjacent.
+
+        Instead of downloading the full trajectory, the user can choose
+        to download only the compressed trajectory file. The following files
+        are set as the default ones, and will be the only ones downloaded if
+        the user does not require the full trajectory at task.get_output()."""
+
+        return [
```

Comment:
Cool, with all these files the post-processing will work.

Reply:
This method gets something, so it should be named `get_…`

Reply:
This method doesnt get something, it sets the default values that will be downloaded in the task.get_outputs

Reply:
I agree with @fabiocruz here, this method is not setting anything: it is just returning a list with filenames, therefore it should be called "get_...". Later you will use it to set something on the task object :
```python
task.set_default_output_files(self.get_default_output_files())
```
and that's fine, but that is the moment when you are "setting" something.
When you invoke this method you are just "getting" a list...